models/attribute-type/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@commercetools-test-data/attribute-type",
+  "version": "4.4.0",
+  "description": "Data model for commercetools API Attribute Type",
+  "bugs": "https://github.com/commercetools/test-data/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/test-data.git",
+    "directory": "models/attribute-type"
+  },
+  "keywords": ["javascript", "typescript", "test-data"],
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/commercetools-test-data-attribute-type.cjs.js",
+  "module": "dist/commercetools-test-data-attribute-type.esm.js",
+  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "dependencies": {
+    "@babel/runtime": "^7.17.9",
+    "@babel/runtime-corejs3": "^7.17.9",
+    "@commercetools-test-data/commons": "4.4.0",
+    "@commercetools-test-data/core": "4.4.0",
+    "@commercetools-test-data/tax-rate": "4.4.0",
```

Comment:
Is TaxRate used in this package?

Reply:
To quote one of my favorite people, "rogue copy pasta strikes again"